src/layouts/DashboardLayout/TopBar.tsx

```diff
@@ -30,7 +30,6 @@ const TopBar = ({
   ...rest
 }) => {
   const classes = useStyles();
-  const [notifications] = useState([]);
```

Comment:
tu bol problem ze tam nie je setter? takze by sme vedeli prepisat AST pri transpilacii, aby to bolo obycajne priradenie do premennej namiesto hooku?

Reply:
To som neskusal